scripts/db/applications.sql

```diff
@@ -1,6 +1,7 @@
 CREATE TABLE applications (
     id SERIAL PRIMARY KEY, 
-    name VARCHAR(255),                  
+    name VARCHAR(255),
+    email VARCHAR(255) UNIQUE NOT NULL,
```

Comment:
Need to adjust as per updated fields :)

Reply:
will be done in https://github.com/catbytes-community/webplatform-backend/issues/10